packages/cli/test/unit/commands/git.test.ts

```diff
@@ -39,11 +39,6 @@ describe('git', () => {
         await expect(client.stderr).toOutput('Found project');
         client.stdin.write('y\n');
 
-        await expect(client.stderr).toOutput(
-          'Do you want to connect "origin" to your Vercel project?'
-        );
-        client.stdin.write('n\n');
```

Comment:
Why was this test changed? This is testing `vc git connect` right?

Reply:
This does test `vc git connect`, which calls `ensureLink` (from the [command's `index.js`](https://github.com/vercel/vercel/blob/main/packages/cli/src/commands/git/index.ts#L91) file), which was calling the git connection logic, which caused this prompt. The reverted logic keeps this within `vc git connect`'s domain, which does not have this prompt.

Reply:
I was expecting `vc git connect` to ask about the remote if there was more than one in the `.git` directory

Reply:
It still does that. In a different test, you can see that behavior being tested: https://github.com/vercel/vercel/blob/main/packages/cli/test/unit/commands/git.test.ts#L324-L327